impeller/docs/babys_first_triangle.md

```diff
@@ -0,0 +1,276 @@
+# Baby's First Triangle
+
+This guide details how to render a triangle using Impeller. We will use the lowest layer of the stack to do this and explore Impellers HAL and shader compilation machinery.
+
+A complete code example of this tutorial is in [`renderer_unittests.cc`](../renderer/renderer_unittests.cc) in `RendererTest.BabysFirstTriangle`.
```

Comment:
the example is a unit test in the impeller source base so it doesn't have the same requirements in terms of includes, linking, actually rendering to the screen, etc; is that worth calling out here?

Reply:
I don't quite follow. Like the unit-tests, the engine TUs that use Impeller have the same requirements. Maybe I am missing something? This guide was a logical successor to the "Setting up a Standalone Impeller in OpenGL ES" guide from last week.